lib/util/sfContext.class.php

```diff
@@ -341,8 +343,8 @@ public function getDatabaseManager()
     /**
      * Retrieve the module directory for this context.
      *
-     * @return string an absolute filesystem path to the directory of the
-     *                currently executing module, if one is set, otherwise null
+     * @return string|null an absolute filesystem path to the directory of the
+     *                     currently executing module, if one is set, otherwise null
```

Comment:
```suggestion
     * @return string|null an absolute filesystem path to the directory of the currently executing module, if one is set, otherwise null
```